src/Compilers/VisualBasic/Portable/Symbols/Source/SourceParameterSymbol.vb

```diff
@@ -266,6 +267,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Friend Overrides Iterator Function GetCustomAttributesToEmit(compilationState As ModuleCompilationState) As IEnumerable(Of VisualBasicAttributeData)
             Dim attributes = MyBase.GetCustomAttributesToEmit(compilationState)
+
+            If Not Location.SourceTree.Options.Features.ContainsKey(InternalSyntax.GetFeatureFlag(InternalSyntax.Feature.CallerArgumentExpression)) Then
```

Comment:
@333fred It looks like `SourceTree` can be null. I'm not sure what to do for the null case, allow the feature to work or emit the attributes as is?

Reply:
We should emit the attribute as is in those cases. I imagine most those will be synthesized parameters, and thus not likely to have this attribute in the first place?